tests/test_brainreg_napari.py

```diff
@@ -78,6 +78,9 @@ def test_workflow(make_napari_viewer, tmp_path):
     assert boundaries.name == "Boundaries"
 
 
+@pytest.mark.xfail(
+    reason="See https://github.com/brainglobe/cellfinder/issues/443"
```

Comment:
Would 
```suggestion
    reason="See https://github.com/brainglobe/cellfinder/issues/443",
    raises=AssertionError
```
be a slight improvement? (have checked the syntax locally, and that this doesn't xfail if I put `raises=ValueError` instead`.

Reply:
Good call, should've just copied it from `cellfinder`!